Innazh/add browser options examples java

innazh opened the pull request:
### **User description**
# Add missing Driver Options examples for Java & incorporate those into the docs

### Description
- Added java examples for Driver Options, specifically: browserName, browserVersion, platformName, acceptInsecureCerts, timeouts (Script Timeout, Page Load Timeout, Implicit Wait Timeout), unhandledPromptBehavior, setWindowRect, strictFileInteractability
- Added the example lines of the code above to the related page in the docs (options*.md files in )

### Motivation and Context
The examples were missing in the docs, I'm replacing the "Add example" badges. 

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [X] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [X] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [X] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.


___

### **PR Type**
Tests, Documentation


___

### **Description**
- Added comprehensive Java test examples for various ChromeOptions settings in `OptionsTest.java`.
- Updated documentation across multiple languages (English, Japanese, Portuguese, Chinese) to include Java code examples with correct line references.
- Replaced placeholder badges in documentation with actual code references to improve clarity.



___



### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Tests</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>OptionsTest.java</strong><dd><code>Add Java test examples for ChromeOptions settings</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java

<li>Added test methods for various ChromeOptions settings.<br> <li> Implemented tests for browserName, browserVersion, and platformName.<br> <li> Included timeout settings and unhandled prompt behavior tests.<br> <li> Added tests for window rect and strict file interactability <br>capabilities.<br>


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1865/files#diff-3eb3ff6008510d0d30b7f1d43c8857bdf29fda42534f08399bccfe1cd323eab5">+89/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>options.en.md</strong><dd><code>Update Java examples in English documentation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/options.en.md

<li>Updated Java code examples with line numbers.<br> <li> Replaced placeholder badges with actual code references.<br>


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1865/files#diff-62bab9f28d27a7f462881a28eca384a0e534a730f849248f396d171820bc748e">+10/-10</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>options.ja.md</strong><dd><code>Update Java examples in Japanese documentation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/options.ja.md

<li>Updated Java code examples with line numbers.<br> <li> Replaced placeholder badges with actual code references.<br>


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1865/files#diff-2ed475220bde1368cfa3883d71db179409ddaf9ccbaea847ed3f6ffe816321e4">+10/-10</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>options.pt-br.md</strong><dd><code>Update Java examples in Portuguese documentation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/options.pt-br.md

<li>Updated Java code examples with line numbers.<br> <li> Replaced placeholder badges with actual code references.<br>


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1865/files#diff-6eaa598a13e9d64c6afbd54b84aef470520557f08c4d183ffb28ceb9ed04d0bb">+10/-10</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>options.zh-cn.md</strong><dd><code>Update Java examples in Chinese documentation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/options.zh-cn.md

<li>Updated Java code examples with line numbers.<br> <li> Replaced placeholder badges with actual code references.<br>


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1865/files#diff-9b23b749355db8154bfcc4a1d70f809b7d4e91a7b3044f9ac74a6c1a005873d6">+10/-10</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

innazh committed:
add browser options test examples in java

added java examples/tests for browserName, browserVersion,platformName,
scriptTimeout, pageLoadTimeout, implicitWaitTimeout,
unhandledPromptBehaviour, setWindowRect, strictFileInteractability

innazh committed:
add java examples to /drivers/options.en.md

added all mising examples for java for driver options page, except for the ones
that require moving code

innazh committed:
rename variables in the added to OptionsTest file functions

innazh committed:
update the line nums in options.en.md
based on the changes I just made to the example file

innazh committed:
update options.ja.md file with java examples for driver options

innazh committed:
update options.pt-br.md file with java examples

innazh committed:
update options.zh-cn.md file with java examples

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=1865) <br/>All committers have signed the CLA.

innazh committed:
remove toString() method from a string var

I was using toString() on a string variable by accident, removing it here.

innazh committed:
addressed the feedback and now using Assertions library

using assertions library, fixed spacing to match the rest of the code,
added the missing imports. All of this threw off the line numbers for
the options docs, so will have to edit it in the next commit.

innazh committed:
Update all line num in the docs

Since I had added imports, the line numbers fro the examples had to be updated.

diemol committed:
Fixing nightly version retrieval

diemol committed:
Merge branch 'trunk' into innazh/add-browser-options-examples-java

diemol reviewed (approved):
Thank you, @innazh!
